RELEASE-NOTES.md

```diff
@@ -14,6 +13,7 @@ It also brings some dreadfully awaited fixes, so be sure to go through the chang
   - 0-length dimensions are now ruled illegal for random variables and raise a `ValueError`.
 - In `sample_prior_predictive` the `vars` kwarg was removed in favor of `var_names` (see [#4327](https://github.com/pymc-devs/pymc3/pull/4327)).
 - Removed `theanof.set_theano_config` because it illegally changed Theano's internal state (see [#4329](https://github.com/pymc-devs/pymc3/pull/4329)).
+- We now depend on `Theano-PyMC` version `1.1.0` or greater (see [#4405](https://github.com/pymc-devs/pymc3/pull/4405)).
```

Comment:
To be precise, this release depends on exactly `Theano-PyMC v1.1.0`.

Reply:
Let's also add a line (above or in "Breaking changes") about the most important changes in `Theano-PyMC`:
```
Major refactorings were done in Theano-PyMC 1.1.0, if you implement custom `Op`s or interact with Theano in any way yourself, make sure to read the [Theano-PyMC 1.1.0 release notes](https://github.com/pymc-devs/Theano-PyMC/releases/tag/rel-1.1.0).
```